docker/build_scripts/build_utils.sh

```diff
@@ -31,27 +31,18 @@ function pyver_dist_dir {
 function do_cpython_build {
     local py_ver=$1
     check_var $py_ver
-    local ucs_setting=$2
-    check_var $ucs_setting
     tar -xzf Python-$py_ver.tgz
     pushd Python-$py_ver
-    if [ "$ucs_setting" = "none" ]; then
-        unicode_flags=""
-        dir_suffix=""
-    else
-        local unicode_flags="--enable-unicode=$ucs_setting"
-        local dir_suffix="-$ucs_setting"
-    fi
-    local prefix="/opt/_internal/cpython-${py_ver}${dir_suffix}"
+    local prefix="/opt/_internal/cpython-${py_ver}"
     mkdir -p ${prefix}/lib
-    ./configure --prefix=${prefix} --disable-shared $unicode_flags > /dev/null
+    ./configure --prefix=${prefix} --disable-shared > /dev/null
     make -j2 > /dev/null
```

Comment:
What about replacing "-j2" to "-j" or "-j$NUM_OF_CPUS" to make use of multiple cpu cores?

AArch64 server systems tend to have 40-200+ cpu cores.

Similar in "do_standard_install" function.